cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -147,6 +147,8 @@ impl RawManifest {
             InitCustomization {
                 profile_common: None,
                 profile_bash: None,
+                profile_fish: None,
```

Comment:
We should mention `profile.fish` and `profile.tcsh` a few lines above

Reply:
Done with commit cbe3b5cad90e03cf1f875654777d439f1480ffca